Sprint-1/exercises/initials.js

```diff
@@ -1,6 +1,7 @@
 let firstName = "Creola";
 let middleName = "Katherine";
 let lastName = "Johnson";
-
+initials = [firstName[0], middleName[0], lastName[0]];
```

Comment:
This would create an array `["C", "K", "J"]` instead of producing a string `"CKJ"`.

Reply:
I am sure in my final code when I tested it, it retuerned CKJ. there is a chance I forgot to actually push the last line commit which will print the "initials" members together.